src/Styleguide/Components/StepSummaryItem.tsx

```diff
@@ -0,0 +1,45 @@
+import { Sans, Serif, space } from "@artsy/palette"
+import React from "react"
+import styled from "styled-components"
+import { BorderBox } from "Styleguide/Elements/Box"
+import { Flex, FlexProps } from "Styleguide/Elements/Flex"
+import { media } from "Styleguide/Elements/Grid"
+
+interface StepSummaryItemProps extends FlexProps {
+  title: string
+  onChange: () => void
+  children: React.ReactNode | null
+}
+
+const Wrapper = styled(BorderBox)`
+  padding: ${space(2)}px;
+  ${media.sm`
+    padding: ${space(3)}px;
+  `};
+  :not(:last-child) {
+    border-bottom: 0;
+  }
+`
+
+export function StepSummaryItem({
+  title,
+  onChange,
+  children,
+  ...others
+}: StepSummaryItemProps) {
+  return (
+    <Wrapper flexDirection="column" width="100%" maxWidth={542} {...others}>
```

Comment:
Should this maxWidth come from somewhere else? It's not really a breakpoint but it feels like it should be part of the design system somehow.

Reply:
Where'd this value come from?

Reply:
Zeplin

Reply:
Ahhh. Be careful with that. Zeplin can be, and often is, wrong. It’s very likely that instead this is intended to align with the grid in some way.

Reply:
Ok for now I've moved the width constraints out of this component, such that the dimensions are controlled by the consumer. I left the widths hard-coded in the story file. They will presumably be constrained by the grid system when in actual use.